src/components/interfaces/MOBILE_API.xml

```diff
@@ -1420,6 +1445,284 @@
+        True: Available, False: Not Available, Not present: Not Available.
+      </description>
+    </param>
+    <param name="stateAvailable" type="Boolean" mandatory="false" >
+      <description>
+        Availability of the getting the Radio state.
+        True: Available, False: Not Available, Not present: Not Available.
+      </description>
+    </param>
+    <param name="signalStrengthAvailable" type="Boolean" mandatory="false" >
+      <description>
+        Availability of the getting the signal strength.
+        True: Available, False: Not Available, Not present: Not Available.
+      </description>
+    </param>
+    <param name="signalChangeThresholdAvailable" type="Boolean" mandatory="false" >
+      <description>
+        Availability of the getting the signal Change Threshold.
+        True: Available, False: Not Available, Not present: Not Available.
+      </description>
+    </param>
+  </struct>
+
+  <enum name="DefrostZone">
+    <element name="FRONT"/>
+    <element name="REAR"/>
+    <element name="ALL"/>
+    <element name="NONE"/>
+  </enum>
+
+  <enum name="VentilationMode">
+    <element name="UPPER"/>
+    <element name="LOWER"/>
+    <element name="BOTH"/>
+    <element name="NONE"/>
+  </enum>
+
+  <enum name="TemperatureUnit">
+    <element name="FAHRENHEIT"/>
+    <element name="CELSIUS"/>
+  </enum>
+
+  <struct name="Temperature">
+    <param name="unit" type="TemperatureUnit" mandatory="true" >
+      <description>Temperature Unit</description>
+    </param>
+    <param name="value" type="Float" mandatory="true" >
+      <description>Temperature Value in TemperatureUnit specified unit. Range depends on OEM and is not checked by SDL.</description>
+    </param>
+  </struct>
+
+   <struct name="ClimateControlData">
+    <param name="fanSpeed" type="Integer" minvalue="0" maxvalue="100" mandatory="false">
+    </param>
+    <param name="currentTemperature" type="Temperature" mandatory="false">
+      <description> Read only parameter.</description>
+    </param>
+    <param name="desiredTemperature" type="Temperature" mandatory="false">
+    </param>
+    <param name="acEnable" type="Boolean" mandatory="false">
+    </param>
+    <param name="circulateAirEnable" type="Boolean" mandatory="false">
+    </param>
+    <param name="autoModeEnable" type="Boolean" mandatory="false">
+    </param>
+    <param name="defrostZone" type="DefrostZone" mandatory="false">
+    </param>
+    <param name="dualModeEnable" type="Boolean" mandatory="false">
+    </param>
+    <param name="acMaxEnable" type="Boolean" mandatory="false">
+    </param>
+    <param name="ventilationMode" type="VentilationMode" mandatory="false">
+    </param>
+  </struct>
+
+<struct name="ClimateControlCapabilities">
+  <description>Contains information about a climate control module's capabilities.</description>
+  <param name="moduleName" type="String" maxlength="50" mandatory="true" >

```

Comment:
This param is not in the accepted proposal. Also why would it be mandatory?

Reply:
@joeygrover module name is not same parameter as others in capabilities, it is short description, and according to https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0071-remote-control-baseline.md it should be mandatory. Do you think it shouldn't?